src/test/java/io/github/opencubicchunks/cubicchunks/cubicgen/TestCustomGeneratorSettingsFixer.java

```diff
@@ -0,0 +1,178 @@
+            "   \"lapisLazuliHeightStdDeviation\":0.125,\n" + 
+            "   \"hillsEmeraldOreSpawnTries\":11,\n" + 
+            "   \"hillsEmeraldOreSpawnProbability\":0.2857143,\n" + 
+            "   \"hillsEmeraldOreSpawnMinHeight\":-Infinity,\n" + 
+            "   \"hillsEmeraldOreSpawnMaxHeight\":-0.5,\n" + 
+            "   \"hillsSilverfishStoneSpawnSize\":7,\n" + 
+            "   \"hillsSilverfishStoneSpawnTries\":7,\n" + 
+            "   \"hillsSilverfishStoneSpawnProbability\":0.25,\n" + 
+            "   \"hillsSilverfishStoneSpawnMinHeight\":-Infinity,\n" + 
+            "   \"hillsSilverfishStoneSpawnMaxHeight\":0.0,\n" + 
+            "   \"mesaAddedGoldOreSpawnSize\":20,\n" + 
+            "   \"mesaAddedGoldOreSpawnTries\":2,\n" + 
+            "   \"mesaAddedGoldOreSpawnProbability\":0.5,\n" + 
+            "   \"mesaAddedGoldOreSpawnMinHeight\":-0.5,\n" + 
+            "   \"mesaAddedGoldOreSpawnMaxHeight\":0.25,\n" + 
+            "   \"heightVariationFactor\":64.0,\n" + 
+            "   \"specialHeightVariationFactorBelowAverageY\":0.25,\n" + 
+            "   \"heightVariationOffset\":0.0,\n" + 
+            "   \"heightFactor\":64.0,\n" + 
+            "   \"heightOffset\":64.0,\n" + 
+            "   \"depthNoiseFactor\":1.024,\n" + 
+            "   \"depthNoiseOffset\":0.0,\n" + 
+            "   \"depthNoiseFrequencyX\":0.0015258789,\n" + 
+            "   \"depthNoiseFrequencyZ\":0.0015258789,\n" + 
+            "   \"depthNoiseOctaves\":16,\n" + 
+            "   \"selectorNoiseFactor\":12.75,\n" + 
+            "   \"selectorNoiseOffset\":0.5,\n" + 
+            "   \"selectorNoiseFrequencyX\":0.016709277,\n" + 
+            "   \"selectorNoiseFrequencyY\":0.008354639,\n" + 
+            "   \"selectorNoiseFrequencyZ\":0.016709277,\n" + 
+            "   \"selectorNoiseOctaves\":8,\n" + 
+            "   \"lowNoiseFactor\":1.0,\n" + 
+            "   \"lowNoiseOffset\":0.0,\n" + 
+            "   \"lowNoiseFrequencyX\":0.005221649,\n" + 
+            "   \"lowNoiseFrequencyY\":0.0026108245,\n" + 
+            "   \"lowNoiseFrequencyZ\":0.005221649,\n" + 
+            "   \"lowNoiseOctaves\":16,\n" + 
+            "   \"highNoiseFactor\":1.0,\n" + 
+            "   \"highNoiseOffset\":0.0,\n" + 
+            "   \"highNoiseFrequencyX\":0.005221649,\n" + 
+            "   \"highNoiseFrequencyY\":0.0026108245,\n" + 
+            "   \"highNoiseFrequencyZ\":0.005221649,\n" + 
+            "   \"highNoiseOctaves\":16\n" + 
+            "}";
+
+    @Before
+    public void setUp() {
+        MinecraftEnvironment.init();
+        CustomCubicMod.LOGGER = LogManager.getLogger("CustomCubicModTest");;
+        CubicBiome.init();
+        CubicBiome.postInit();
+    }
+
+    @Test
+    public void testCustomGeneratorSettingsFixer() {
+        CustomGeneratorSettings settings = CustomGeneratorSettings.fromJson(cc655ExamplePreset);
+        assertEquals(13,settings.standardOres.size());
```

Comment:
It doesn't verify that the preset actually has all the right data

Reply:
So what I should assert?